### PR TITLE
Add support for skipping generating stack traces.

### PR DESCRIPTION
**Pull request by ruiaraujo**
Stack generation is an expensive operation that might be skippable for certain scenarios.

### CODE REVIEW

File: src/execution.ts [hunk 1]
```diff
@@ -775,7 +789,7 @@ function unpromisify(
         value
             .then(
                 (res: any) => cb(res, null),
-                (err: Error) => (err != null ? cb(null, err) : cb(null, new Error("")))
+                (err: Error) => (err != null ? cb(null, err) : cb(null, new (CustomGraphQLError as any)("")))
```
Comment (topicus) [inline src/execution.ts]: You can do something like:
```
class CustomGraphQLError {
  constructor(message) {
    this.message = message;
    this.name = "CustomGraphQLError";
    return Object.setPrototypeOf(this, Error.prototype);
  }
  toString() {
    return `${this.name}: ${this.message}`;
  }
}
```
then:

```
const e = new CustomGraphQLError("Some error");
console.log(e instanceof Error) // true
```
The performance penalty is minimal.

Comment (boopathi) [reply src/execution.ts]: Use `GraphqlJitError`. or `GraphQLError`

File: src/execution.ts [hunk 2]
```diff
@@ -55,6 +55,11 @@ export interface CompilerOptions {
     // only safe for use if the output is completely correct.
     disableLeafSerialization: boolean;
 
+    // Disable builtin scalars and enum serialization
+    // which is responsible for coercion,
+    // only safe for use if the output is completely correct.
+    captureStackErrors: boolean;
```
Comment (vigneshshanmugam) [inline src/execution.ts]: nit: comments is unrelated to the flag.

Comment (ruiaraujo) [reply src/execution.ts]: not a nit. Good catch :+1:

**Comment by ruiaraujo** [comment]
:+1:

**Comment by topicus** [comment]
:+1:

**Pull request merged**